mods/wlssb/statuses.js

```diff
@@ -99,17 +87,16 @@ exports.BattleStatuses = {
 	mystifi: {
 		exists: true,
 		onStart: function () {
-			this.add('c', '~Mystifi', '__I\'ll HM01 u faster then sanic m89__');
```

Comment:
Mystifi~

Reply:
%Lycanium Z: Wait Mystifi is admin again?
~HoeenHero: that was temp
%Lycanium Z: Oh